examples/langchain/README.md

```diff
@@ -89,4 +97,53 @@ query using the `marklogic_contextual_query_retriever.py` module in this project
 
 This retriever builds a term-query using words from the question. Then the term-query is
 added to the structured query and the merged query is used to select from the documents 
-loaded via `load_data.py`.
\ No newline at end of file
+loaded via `load_data.py`.
+
+## Testing using MarkLogic 12EA BM25 scoring and Vector functionality
+
+### MarkLogic 12EA Setup
+
+To try out this functionality out, you will need acces to an instance of MarkLogic 12
+(currently internal or Early Access only). You may use docker 
+[docker-compose](https://docs.docker.com/compose/) to instantiate a new MarkLogic
+instance with port 8003 available (you can use your own MarkLogic instance too, just be
+sure that port 8003 is available):
+
+    docker-compose -f docker-compose-12.yml up -d --build
+
+### Deploy With Gradle
+
+You will also need to deploy the application. However, for this example, you will need
+to include an additional switch on the command line to deploy a TDE schema that takes
+advantage of the vector capabilities in MarkLogic 12.
+
+    ./gradlew -i mlDeploy -PmlSchemasPath=src/main/ml-schemas-12
```

Comment:
This works, but you can also make a custom task to make it even nicer - e.g. `loadMarkLogic12Schemas`. I can add that later.